pkg/kv/kvserver/replica_rankings.go

```diff
@@ -138,23 +143,33 @@ func (rr *ReplicaRankings) TopLoad() []CandidateReplica {
 // prevents concurrent loaders of data from messing with each other -- the last
 // `update`d accumulator will win.
 type RRAccumulator struct {
-	dim rrPriorityQueue
+	dims map[load.Dimension]*rrPriorityQueue
```

Comment:
The map here feels heavyweight when we could use a slice like we do in `load.Vector`. It doesn't need to be changed here, but consider replacing on master.

Reply:
Agree - I will change on master.

Reply:
Perhaps you could try out some generics here with a structure like `type load.Map[T any] [nDimensions]T`.